addons/api/addon/models/role.js

```diff
@@ -42,6 +51,61 @@ export default class RoleModel extends GeneratedRoleModel {
       : null;
   }
 
+  /**
+   * Convenience for looking up the grant scopes, if loaded.
+   */
+  get grantScopes() {
+    // filter out scopes that start with an o_ or p_
+    // then sort the keywords
+    const grantScopes = [];
+    if (this.grant_scope_ids) {
+      const keywordIDs = this.grant_scope_ids.filter((id) => {
+        return !id.startsWith('o_') && !id.startsWith('p_') && id !== 'global';
+      });
```

Comment:
Why not just use the keywords from `GRANT_SCOPE_KEYWORDS` to find them rather than doing the inverse?

Reply:
Because I got lost in the woods refactoring this a few times. Thanks for the good suggestion. Updated.